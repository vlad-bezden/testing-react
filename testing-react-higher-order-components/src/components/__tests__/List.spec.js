import React from 'react'
import renderer from 'react-test-renderer'
import List from '../List'

test('Should show the gists', () => {
  const gists = [{ id: 1, description: 'description' }]
  const component = renderer.create(<List data={gists} />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})