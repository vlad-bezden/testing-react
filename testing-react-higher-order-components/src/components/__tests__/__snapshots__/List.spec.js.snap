// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should show the gists 1`] = `
<ul>
  <li>
    description
  </li>
</ul>
`;
